registry.k8s.io/images/k8s-staging-cluster-api/images.yaml

```diff
@@ -81,6 +81,7 @@
     "sha256:c8ce4e6ab4d81858022a2844ae7ecc199d659323c06e926c2663be2ab0db0e67": ["v1.1.4"]
     "sha256:df8dca5656e3440e6b37f4a2de44a35c4ba20a3b48a0294703f6785365c9f6f5": ["v1.1.5"]
     "sha256:763eac18e8b16f89a8e36f19b052d120231f6ef73f5bbab86457c4d83baa1ce3": ["v1.1.6"]
+    "sha256:36ca3c30a6aef69070371baf0731cf38ce82221c4695f312352dbaa37710f3a8": ["v1.10.0"]
```

Comment:
`36ca3c30a6ae` matches the SHA for cluster-api-controller v1.10.0 in the [staging repository](https://console.cloud.google.com/gcr/images/k8s-staging-cluster-api/global/cluster-api-controller@sha256:36ca3c30a6aef69070371baf0731cf38ce82221c4695f312352dbaa37710f3a8/details). 👍🏻